analysis/import/SonarImporter/src/main/java/de/maibornwolff/codecharta/importer/sonar/dataaccess/SonarMeasuresAPIDatasource.java

```diff
@@ -61,7 +68,25 @@ public SonarMeasuresAPIDatasource(String user, URL baseUrl, String projectKey) {
         this.projectKey = projectKey;
     }
 
-    public Measures getMeasures(List<String> metrics, int pageNumber) throws SonarImporterException {
+    public ComponentMap getComponentMapFromDS(List<String> metricsList) {
```

Comment:
why do you put DS (DataSource) into the method name? Isn't that already clear from the class name?